Gruntfile.js

```diff
@@ -175,6 +178,13 @@ module.exports = function(grunt) {
             cwd: '<%= config.bootstrap.dist %>',
             src: ['js/bootstrap.min.js'],
             dest: '<%= config.dist %>/assets'
+          },
+          // include ibm plex
+          {
+            expand: true,
+            cwd: '<%= config.plex.dist %>',
+            src: ['IBM-Plex-Sans/fonts/complete/**/*'],
```

Comment:
It's missing `IBM-Plex-Mono`.

```Suggestion
            src: ['IBM-Plex-Sans/fonts/complete/**/*', 'IBM-Plex-Mono/fonts/complete/**/*'],
```

Reply:
My bad :).

The font was not used yet, hence I forgot to add it.

Reply:
![image](https://user-images.githubusercontent.com/21984219/152120743-4e44df44-71e9-400a-b9d7-74bbcfacbe0f.png)

Indeed, it isn't... I'll look into changing the default monospace font to Plex

Reply:
Did it already! https://github.com/bpmn-io/bpmn.io/pull/118